src/test/java/com/google/devtools/build/lib/runtime/BlazeOptionHandlerTest.java

```diff
@@ -1084,6 +1092,93 @@ public void testParseOptions_2LongChains() {
                     + "counted twice and may lead to unexpected behavior."));
   }
 
+  @Test
+  public void testParseOptions_invocationPolicy() throws Exception {
+    setUp(makeInvocationPolicy());
```

Comment:
is it possible to inline some of the details of our made-up invocation policy in the test method? this will make it more clear what we have coverage for, as well as provide more immediate context for the assertions. we can still have factory helpers for the boilerplate to improve readability. for example...

```
  setUp(
      InvocationPolicy.newBuilder()
          .addFlagPolicy(setValue("parent", "parent_test_string", "parent-for-parent", ALLOW_OVERRIDES))
         .addFlagPolicy(setValue(...))
         ...
```

Reply:
Inlined, and added comments to each FlagPolicy explaining why it's there and what's special about it.